charts/retool/values.yaml

```diff
@@ -299,6 +299,15 @@ jobRunner:
   # will already launch a job runner pod
   # enabled: true
 
+  # These resource apply only to the jobs-runner pod. Set the resources to scale jobs-runner pod separately from main-backend pods.
```

Comment:
nit reword:

If necessary, specify the resources to provision the jobRunner pod separately from the main backend pods.